src/inference/hand_detection.py

```diff
@@ -1,14 +1,16 @@
 #!/usr/bin/env python3
 
 import cv2
-
-from yolo import YOLO
+import numpy as np
+from inference.yolo import YOLO
 
 
 class HandDetector:
 
-    def __init__(self, image_size=416, confidence=0.2):
-        self.yolo = YOLO("models/cross-hands-tiny.cfg", "models/cross-hands-tiny.weights", ["hand"])
+    def __init__(self, image_size, confidence, model_dirs):
+
+        self.yolo = YOLO(model_dirs+"cross-hands-tiny.cfg", model_dirs+"cross-hands-tiny.weights",
```

Comment:
what if the user forgot to add `/` at the end of the model_dirs string?

Reply:
I don't know how to create more robust solution, `/` has to be somewhere.